apps/allocations/app/components/App/App.js

```diff
@@ -122,6 +113,30 @@ const App = () => {
       </IdentityProvider>
     </Main>
   )
+
+  Wrap.propTypes = {
+    children: PropTypes.node.isRequired,
+  }
+
+  if (accounts.length === 0) {
+    return <Wrap><Empty action={onNewAccount} /></Wrap>
+  }
+
+  return (
+    // TODO: Profile App with React.StrictMode, perf and why-did-you-update, apply memoization
+    <Wrap>
+      <Header
+        primary="Allocations"
+        secondary={
+          <Button mode="strong" icon={<IconPlus />} onClick={onNewAccount} label="New Account" />
```

Comment:
This button is not present anymore in the designs, we should clarify if it is still needed, on that case, reflect the clarification in the PR comment for the reviewer to know. Otherwise, please remove the button

Reply:
This PR is focused on the No budget page, in which the whole Header is not present anymore. In that sense, I believe it's out of scope to deal with that button here.

But in any case, the button is still present in the designs, except it's hidden inside the Actions menu.

Reply:
Ok, I think this fits in the final checking task before mainnet deployment, where we could fine tune these kind of cosmetic details 👍